### DISCUSSION TITLE
Is this integration still alive?

### DISCUSSION THREAD

**Discussion by nagtegaal** [binsentsu/home-assistant-solaredge-modbus#249] [General]
Is this integration still alive and still having maintainers?
The warning in the Home Assistant logs about the [async_forward_entry_setup](https://github.com/binsentsu/home-assistant-solaredge-modbus/issues/239) is most annoying, and there is a fix waiting in the thread is referred to. Could this be committed and be released as a new version please, if there is anyone who tries to maintain this integration?

**Comment by mpredfearn** [comment]
Personally I shifted over to https://github.com/WillCodeForCats/solaredge-modbus-multi which is more actively maintained and has more features.
